types/react-syntax-highlighter/index.d.ts

```diff
@@ -84,6 +93,11 @@ declare module 'react-syntax-highlighter/dist/esm/prism' {
     export default class SyntaxHighlighter extends React.Component<SyntaxHighlighterProps> {}
 }
 
+declare module 'react-syntax-highlighter/dist/esm/create-element' {
+    function createElement(props: any): JSX.Element;
+    export default createElement;
+}
```

Comment:
A couple things here:
1. Look at the definition of `react-syntax-highlighter/dist/esm/prism` before this and use that as a reference. Similar to how it is using `SyntaxHighlighterProps`, I think you should define a `CreateElementProps` and use that.
2. A module should also be declared for the `cjs` version.